tests/test_prefect_flows.py

```
@@ -356,3 +359,79 @@ def test_v4_uses_sequential_when_flag_false(mock_sequential, mock_tolerant):
+    returning False cleanly is preferable to leaning on that fallback."""
+    task = MagicMock()
+    task_run = MagicMock()
+    task_run.start_time = "not-a-datetime"
+    task_run.name = "dbtjob-dbt-run"
+    state = MagicMock()
+    state.timestamp = "also-not-a-datetime"
+    # We don't assert the return value here — we only assert it doesn't bubble
+    # an exception out of the helper itself in any way that surprises us.
+    try:
+        result = _retry_if_short_runtime(task, task_run, state)
+    except Exception:  # pylint: disable=broad-except
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)